### DISCUSSION TITLE
Where is code for the frame output?

### DISCUSSION THREAD

**Discussion by philippneugebauer** [ultralytics/yolov5#5633] [Q&A]
I tried to find the prefix part for the terminal output, but I couldn't find it:
video 1/1 (1/900) /usr/src/app/images/yolo_input.mp4: 192x640 Done. (0.036s)

Where is the string of `video 1/1 (1/900) /usr/src/app/images/yolo_input.mp4` constructed?

**Comment by glenn-jocher** [comment]
@philippneugebauer probably here:
https://github.com/ultralytics/yolov5/blob/7473f0f95dbc9ef9dd1706274906c99eac2ee2f9/utils/datasets.py#L220

or here:
https://github.com/ultralytics/yolov5/blob/7473f0f95dbc9ef9dd1706274906c99eac2ee2f9/utils/datasets.py#L213
